store/map/index.js

```diff
@@ -160,6 +172,12 @@ export const actions = {
 }
 
 export const getters = {
+  getThemes(state) {
```

Comment:
why not in themes? Keep your store clean with [modules](https://vuex.vuejs.org/guide/modules.html). Or don't try to put everything in the store.

Reply:
moved it to themes.js